app/javascript/pages/fullArticle.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import { Text } from '@patternfly/react-core';
+import colors from '../utils/colors';
+import previewImage from '../../assets/images/img1.png';
+const styles = {
+  title: {
+    textAlign: 'center',
+    alignSelf: 'center',
+    color: colors.appBlue,
+    fontSize: '50px',
+    fontWeight: 'bolder',
+    marginInline: '10%'
+  },
+  img: {
+    marginInline: '25%',
+    marginTop: '2%',
+    marginBottom: '2%',
+    borderRadius: 10
+  },
+  body: {
+    textAlign: 'center',
+    alignSelf: 'center',
+    color: colors.appGrey,
+    fontSize: '30px',
+    marginInline: '20%'
+  }
+};
+const FullArticle = () => {
+  const { state } = useLocation();
```

Comment:
IMHO it would be better to fetch the data from an API call, in foreman we use the `axios` library.